tutorials/platform/web/customizing_html5_shell.rst

```diff
@@ -74,6 +74,9 @@ The following optional placeholders will enable some extra features in your cust
 - ``$GODOT_SPLASH``:
   Path to an image used as the boot splash as defined in the Project Settings.
 
+- ``$GODOT_SPLASH_COLOR``
+  The splash screen background color as defined in the Project Settings, converted to a hex color code.
```

Comment:
Should link the the project setting directly, either:
```suggestion
  The splash screen background color as defined in 
  :ref:`Project Settings > Application > Boot Splash > BG Color <class_ProjectSettings_property_application/boot_splash/bg_color>`,
  converted to a hex color code.
```
or
```suggestion
  The splash screen background color as defined in the
  :ref:`BG Color <class_ProjectSettings_property_application/boot_splash/bg_color>`
  setting in **Project Settings > Application > Boot Splash**, converted to a hex color code.
```

Long links aren't split line-by-line so the second option renders better sometimes.

Reply:
Good idea! Should I then also change it for `$GODOT_PROJECT_NAME` and `$GODOT_SPLASH` in this PR?

Reply:
That would be a great change if you want to do it! I just didn't suggest it because I didn't want to ask you do work outside of the scope of this change.

Reply:
Done! I'm more than happy to put this small bit of additional work into it if it makes the engine and its docs better :). Should I also squash the commits into one?

Reply:
Yes, please squash if you are able to.

On the docs repo it's not a big deal to have multiple commits in a PR, since we also use GitHub's "Squash and merge" UI sometimes. But if you can, it's still better to stick to a single commit using either interactive rebase and/or an amend workflow.

Reply:
Done! Please let me know if there are any other changes I should make.